.vscode/tasks.json

```diff
@@ -13,10 +13,11 @@
             "problemMatcher": "$tsc-watch",
             "isBackground": true,
             "presentation": {
-                "reveal": "never"
+                "reveal": "silent"
```

Comment:
What is presentation and what does this do?

Reply:
When you hit F5 in VSCode it runs these tasks. Presentation controls whether it brings the terminal to the foreground or not. By changing it to "silent" i'm saying, only bring it to the foreground if there are errors.